Frontend/ui-driver/src/Flow.purs

```diff
@@ -1707,16 +1707,20 @@ aadhaarVerificationFlow = do
   out <- UI.aadhaarVerificationScreen
   case out of
     ENTER_AADHAAR_OTP state -> do
+      void $ lift $ lift $ loaderText (getString VALIDATING) (getString PLEASE_WAIT_WHILE_IN_PROGRESS)
+      void $ lift $ lift $ toggleLoader true
       res <- lift $ lift $ Remote.triggerAadhaarOtp state.data.aadhaarNumber
       case res of 
         Right (GenerateAadhaarOTPResp resp) -> do
+          void $ lift $ lift $ toggleLoader false
```

Comment:
We could have made it false before case statement itself just after API call. No need to write same line twice